{
  "uid" : "aab9e294db964d47",
  "name" : "登录测试反向用例",
  "fullName" : "cases.test_login.TestLogin#test_login_failed",
  "historyId" : "28fbee3f7849e5c692210a9f12e7fe96",
  "time" : {
    "start" : 1622204095344,
    "stop" : 1622204097377,
    "duration" : 2033
  },
  "status" : "broken",
  "statusMessage" : "KeyError: 'error'",
  "statusTrace" : "self = <cases.test_login.TestLogin object at 0x00000275E5958C18>\nhandler = (<appium.webdriver.webdriver.WebDriver (session=\"06a3f7eb-deac-4925-8b99-b26a0d0d1062\")>, <pages.page_action.PageAction object at 0x00000275E597F198>)\ndata = {'error_msg': '登录验证失败，用户名不存在', 'name': '反向用例-用户名错误-密码正确', 'password': 'ipmnlfiof', 'username': 'plkjf才有'}\n\n    @allure.story('钓鱼人APP自动化测试反向用例测试')\n    @allure.title('登录测试反向用例')\n    @pytest.mark.parametrize('data', failed_data)\n    def test_login_failed(self, handler, data):\n        logger.info('开始执行反向测试用例：{}'.format(data['name']))\n        handler[1].login(data['username'], data['password'])\n        logger.info('反向用例输入完成')\n>       assert handler[1].is_toast_show(data['error'])\nE       KeyError: 'error'\n\ncases\\test_login.py:48: KeyError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "handler",
    "time" : {
      "start" : 1622204065284,
      "stop" : 1622204084024,
      "duration" : 18740
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : false,
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0
  }, {
    "name" : "prepare_to_login",
    "time" : {
      "start" : 1622204084024,
      "stop" : 1622204093721,
      "duration" : 9697
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : false,
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0
  } ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "KeyError: 'error'",
    "statusTrace" : "self = <cases.test_login.TestLogin object at 0x00000275E5958C18>\nhandler = (<appium.webdriver.webdriver.WebDriver (session=\"06a3f7eb-deac-4925-8b99-b26a0d0d1062\")>, <pages.page_action.PageAction object at 0x00000275E597F198>)\ndata = {'error_msg': '登录验证失败，用户名不存在', 'name': '反向用例-用户名错误-密码正确', 'password': 'ipmnlfiof', 'username': 'plkjf才有'}\n\n    @allure.story('钓鱼人APP自动化测试反向用例测试')\n    @allure.title('登录测试反向用例')\n    @pytest.mark.parametrize('data', failed_data)\n    def test_login_failed(self, handler, data):\n        logger.info('开始执行反向测试用例：{}'.format(data['name']))\n        handler[1].login(data['username'], data['password'])\n        logger.info('反向用例输入完成')\n>       assert handler[1].is_toast_show(data['error'])\nE       KeyError: 'error'\n\ncases\\test_login.py:48: KeyError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "5e76eaf298caf7ac",
      "name" : "log",
      "source" : "5e76eaf298caf7ac.txt",
      "type" : "text/plain",
      "size" : 4012
    } ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : true,
    "shouldDisplayMessage" : true,
    "attachmentsCount" : 1
  },
  "afterStages" : [ {
    "name" : "handler::0",
    "time" : {
      "start" : 1622204101990,
      "stop" : 1622204102530,
      "duration" : 540
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : false,
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0
  } ],
  "labels" : [ {
    "name" : "feature",
    "value" : "钓鱼人APP自动化测试-登陆功能测试"
  }, {
    "name" : "story",
    "value" : "钓鱼人APP自动化测试反向用例测试"
  }, {
    "name" : "epic",
    "value" : "钓鱼人APP自动化测试"
  }, {
    "name" : "tag",
    "value" : "@pytest.mark.usefixtures('prepare_to_login')"
  }, {
    "name" : "parentSuite",
    "value" : "cases"
  }, {
    "name" : "suite",
    "value" : "test_login"
  }, {
    "name" : "subSuite",
    "value" : "TestLogin"
  }, {
    "name" : "host",
    "value" : "DESKTOP-CIG6MR7"
  }, {
    "name" : "thread",
    "value" : "13708-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "cases.test_login"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "data",
    "value" : "{'name': '反向用例-用户名错误-密码正确', 'username': 'plkjf才有', 'password': 'ipmnlfiof', 'error_msg': '登录验证失败，用户名不存在'}"
  } ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ "@pytest.mark.usefixtures('prepare_to_login')" ]
  },
  "source" : "aab9e294db964d47.json",
  "parameterValues" : [ "{'name': '反向用例-用户名错误-密码正确', 'username': 'plkjf才有', 'password': 'ipmnlfiof', 'error_msg': '登录验证失败，用户名不存在'}" ]
}